client/src/components/pages/home/HomeBanner.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+
+import ContentManager from "../../../ContentManager";
+import Content from "../../../content/home-en";
+import { get } from "../../../utilities";
+
+import stripesBottomLeft from "./svg/stripes-bottom-left.svg";
+import stripesTopRight from "./svg/stripes-top-right.svg";
+import stripedCircleWhite from "./svg/striped-circle-white.svg";
+import stripedCircleOrange from "./svg/striped-circle-orange.svg";
+
+import "./HomeBanner.css";
+import { Button, Typography } from "antd";
+import { useMatchMedia } from "../../../utilities";
+
+const UI = ContentManager.getUI();
+
+/**
+ * @typedef {Object} LoginProps
+ *
+ * @property {Object} user
+ * @property {Function} setUser
+ */
+/**
+ * Opens a popup window to osu! OAuth login.
+ * This is copied from LoginButton.js
+ *
+ * @param {LoginProps} props
+ */
+async function login({ user, setUser }) {
+  if (user.username) {
+    await fetch("/auth/logout");
+    setUser({});
+    return;
+  }
+
+  const width = 600;
+  const height = 600;
+  const left = window.innerWidth / 2 - width / 2;
+  const top = window.innerHeight / 2 - height / 2;
+
+  const popup = window.open(
+    "/auth/login/",
+    "",
+    `toolbar=no, location=no, directories=no, status=no, menubar=no,
+    scrollbars=no, resizable=no, copyhistory=no, width=${width},
+    height=${height}, top=${top}, left=${left}`
+  );
+
+  // Not sure why but onclose doesn't work
+  const loop = setInterval(async () => {
+    if (popup.closed) {
+      clearInterval(loop);
+      const userData = await get("/api/whoami");
+      setUser(userData);
+    }
+  }, 50);
+  return loop;
+}
+
+function HomeBannerBackground() {
+  return (
+    <div className="HomeBannerBackground-container">
+      <img src={stripesBottomLeft} className="HomeBannerBackground-stripes-left" />
+      <img src={stripesTopRight} className="HomeBannerBackground-stripes-right" />
+      <img src={stripedCircleWhite} className="HomeBannerBackground-striped-circle-white" />
+      <img src={stripedCircleOrange} className="HomeBannerBackground-striped-circle-orange" />
+    </div>
+  );
+}
+
+/**
+ * @param {LoginProps} props
+ */
+function LoginButtons({ user, setUser }) {
+  return (
+    <div className="HomeBanner-login-buttons-container">
+      <Button
+        className="cta"
+        size="large"
+        onClick={() => {
+          login({ user, setUser });
+        }}
+      >
+        {user?.username ? UI.logout : UI.login}
+      </Button>
+      <Button className="outlined-light" size="large">
+        Learn More
```

Comment:
i dont know what we'd use this for lol
tbh i'd consider just hiding this component entirely for now, since this button doesn't do anything and the login button is also kinda redundant

Reply:
Removed in 400d747fcda341fb788f49c155bcf464530e1af2
This is how it looks after that:
![image](https://github.com/cory2067/gtsosu/assets/5846322/ba207795-8319-4f85-bcc7-fc9450dd8634)